### PR TITLE
Bump sauce-testrunner-utils@1.0.0

### PR DESCRIPTION
**Pull request by FriggaHel**
- Bump sauce-testrunner-utils@1.0.0
- Removed unused `demo-kitchen-sink`

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -52,6 +52,9 @@ jobs:
       - name: Install Webkit Dependencies
         run: npx playwright install-deps webkit
 
+      - name: Get NPM path
+        run: echo "NPM_CLI_PATH=`which npm | sed 's/bin\/npm/lib\/node_modules\/npm\/bin\/npm-cli.js/'`" >> $GITHUB_ENV
```
Comment (alexplischke) [inline .github/workflows/test.yml:56]: Why is this one necessary, but not for the other runners? 🤔

Comment (FriggaHel) [reply .github/workflows/test.yml:56]: That's what I'm trying to understand currently 😞

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -168,6 +171,11 @@ jobs:
         run: |
           saucectl run --config .sauce/config_on_sauce.yml --runner-version "url: https://storage.googleapis.com/${{ secrets.GCS_RUNNER_BUCKET }}/sauce-cypress-runner-${{ github.run_id }}.zip"
 
+      - name: post-release Tests
```
Comment (alexplischke) [inline .github/workflows/test.yml:174]: We shouldn't have to run two post-release tests though. The previous step above is doing that already. If there's a difference in configuration, we should perhaps seek to combine 🙏

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -243,6 +251,11 @@ jobs:
         run: |
           npx saucectl run --config .sauce/config_on_sauce_mac.yml --runner-version "url: https://storage.googleapis.com/${{ secrets.GCS_RUNNER_BUCKET }}/sauce-cypress-macos-${{ github.run_id }}.zip"
 
+      - name: post-release Tests
```
Comment (alexplischke) [inline .github/workflows/test.yml:254]: See comment https://github.com/saucelabs/sauce-cypress-runner/pull/223/files#r1127197862 as well.

**Review by alexplischke — APPROVED** [review]

**Pull request merged**